src/nc/input/le/LeParser.cpp

```diff
@@ -0,0 +1,346 @@
+#include "LeParser.h"
+#include <nc/core/input/ParseError.h>
+#include <nc/core/input/Utils.h>
+#include <nc/core/image/Section.h>
+#include <nc/core/image/Image.h>
+#include <nc/core/image/Relocation.h>
+#include <nc/common/make_unique.h>
+#include <nc/common/LogToken.h>
+#include <nc/common/ByteOrder.h>
+#include <nc/common/Foreach.h>
+
+namespace nc {
+namespace input {
+namespace le {
+
+namespace {
+
+struct header_pos {
+    qint64 mz;
+    qint64 le;
+};
+
+using nc::core::input::read;
+using nc::core::input::ParseError;
+
+const ByteOrder bo = ByteOrder::LittleEndian;
+
+header_pos find_header_pos(QIODevice *in) {
+    char a;
+    while (in->getChar(&a)) {
+    again:
+        if (a != 'M')
+            continue;
+        char b;
+        if (!in->getChar(&b))
+            break;
+        if (b != 'Z') {
+            a = b;
+            goto again;
+        }
+        qint64 mz = in->pos() - 2;
+        if (!in->seek(mz + 0x3c))
+            break;
+        std::uint32_t le_off;
```

Comment:
MSVC 2010 does not support fixed-width types in std:: namespace.
Sorry, I misguided you when asked to add std:: prefix (starting to forget MSVC 2010 issues).